bodhi/server/templates/update.html

```diff
@@ -365,26 +334,17 @@
         dataType: 'jsonp',
         cache: false,
         success: receive_resultsdb,
-        error: function(v1, v2, v3) {
-          activeResultsAjaxCalls--;
-          $('#resultsdb .spinner').remove();
-          $('#resultsdb').append(
-            '<h4 class="text-danger">Error getting results.</h4>');
-        },
       });
     };
 
     $("#tab-automatedtests").append(" <span class='fa fa-spinner fa-spin'></span>");
 
-    for (index = 0, len = builds.length; index < len; ++index) {
-      request_results(base_url+"results/latest?original_spec_nvr="
-        +builds[index]+"&testcases=org.centos.prod.ci.pipeline.complete");
-      request_results(base_url+"results/latest?item="+builds[index]+"&type=koji_build&testcases:like=dist.*");
-    }
+    get_greenwave_information({
+        'product_version': '${update.product_version}',
+        'decision_context': 'bodhi_update_push_stable',
```

Comment:
In the Bodhi backend code, there is also a query to Greenwave to assert that it's OK to push an update to testing when the update is pending. Should we do that here as well?

Reply:
Hm, yes.

Reply:
See f2a4919b82c685ce9038e1bd536bec214b928e32.